src/flicking.js

```diff
@@ -93,7 +93,8 @@ eg.module("flicking", ["jQuery", eg, window, document, eg.MovableCoord], functio
 		construct: function (element, options, _prefix) {
 			this.$wrapper = $(element);
 
-			if (!this.$wrapper[0] || !this.$wrapper[0].hasChildNodes()) {
+			var children = this.$wrapper.children();

```

Comment:
Not publicly open yet, but we have convention that for variable name of DOM elements should have '$' prefix.
So plz, update `children` to `$children`.

BTW, the detailed code convention will be available as soon as possible.